packages/website/components/mdx/code/block.tsx

```diff
@@ -1,10 +1,10 @@
 import * as React from 'react';
 import { AkCodeBlock } from '@atlaskit/code';
 
-interface Props {
+export type Props = {
```

Comment:
why type and not interface? I've seen interface used in the typescript docs for prop types

Reply:
I haven't heard a lot about why either pattern is good/bad. May switch this later if it's not standard.

Reply:
From my limited TS experience `interface` seems like what you should use by default. `type` gets used when you want to do funky type stuff